src/EnergyPlus/SimulationManager.cc

```diff
@@ -1190,6 +1190,14 @@ namespace SimulationManager {
             }
         }
 
+        ObjexxFCL::gio::write(OutputFileInits, fmtA) << "! <PerformancePrecisionTradeoffs:DirectSolution>, Simulation {{Yes/No}}, Direct Solution";
+        if (DoCoilDirectSolutions) {
+            ObjexxFCL::gio::write(OutputFileInits, fmtA) << " Direct Solution Simulation, Yes";
+            ShowWarningError("PerformancePrecisionTradeoffs: Direct Solution Simulation is selected.");
```

Comment:
As this object grows, I'm not sure if we want to have specific warnings for each option, or just write a generic warning if one or more tradeoff options are active? Something like 
"PerformancePrecisionTradoffs: One or more performance overrides have been selected, see the eio output for more details." 
@JasonGlazer @xuanluo113 ?